tron/core/actionrun.py

```diff
@@ -483,7 +484,11 @@ def state_data(self):
 
     def render_command(self):
         """Render our configured command using the command context."""
-        return self.bare_command % self.context
+        try:
+            parse_str = self.bare_command % self.context
+            return StringFormatter(self.context).format(parse_str)
```

Comment:
Can you help explain how we will deploy this code safely, given that we have existing `{}`'s?
If we ship this code, won't it break right away on the json stuff?
Then we will get an exception (:409) and then do the old behavior, which is fine.
Should be good to go, right?

It makes me wonder how we ever protected ourselves from exceptions before? idk.

Reply:
Yeah, you're right. If anything (exception) happens here, we would just fall back to the old behavior. So I think json stuff should be fine.